client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -63,66 +77,43 @@ class JetpackChecklist extends PureComponent {
 							"We've automatically protected you from brute force login attacks."
 						) }
 					/>
-					<Task completed title={ translate( "We've automatically turned on spam filtering." ) } />
-					<Task
-						completed
-						completedButtonText={ translate( 'Change' ) }
-						completedTitle={ translate( 'You turned on backups and scanning.' ) }
-						description={ translate(
-							"Connect your site's server to Jetpack to perform backups, rewinds, and security scans."
-						) }
-						duration={ translate( '%d minute', '%d minutes', { count: 2, args: [ 2 ] } ) }
-						onClick={ this.handleTaskStart( {
-							taskId: 'jetpack_backups',
-							url: `/activity-log/${ siteSlug }`,
-						} ) }
-						title={ translate( 'Backups & Scanning' ) }
-					/>
-					<Task
-						completed={ this.isComplete( 'jetpack_monitor' ) }
-						completedButtonText={ translate( 'Change' ) }
-						completedTitle={ translate( 'You turned on Jetpack Monitor.' ) }
-						description={ translate(
-							"Monitor your site's uptime and alert you the moment downtime is detected with instant notifications."
-						) }
-						duration={ translate( '%d minute', '%d minutes', { count: 3, args: [ 3 ] } ) }
-						onClick={ this.handleTaskStart( {
-							taskId: 'jetpack_monitor',
-							tourId: 'jetpackMonitoring',
-							url: `/settings/security/${ siteSlug }`,
-						} ) }
-						title={ translate( 'Jetpack Monitor' ) }
-					/>
-					<Task
-						completed={ this.isComplete( 'jetpack_plugin_updates' ) }
-						completedButtonText={ translate( 'Change' ) }
-						completedTitle={ translate( 'You turned on automatic plugin updates.' ) }
-						description={ translate(
-							'Choose which WordPress plugins you want to keep automatically updated.'
-						) }
-						duration={ translate( '%d minute', '%d minutes', { count: 3, args: [ 3 ] } ) }
-						onClick={ this.handleTaskStart( {
-							taskId: 'jetpack_plugin_updates',
-							tourId: 'jetpackPluginUpdates',
-							url: `/plugins/manage/${ siteSlug }`,
-						} ) }
-						title={ translate( 'Automatic Plugin Updates' ) }
-					/>
-					<Task
-						completed={ this.isComplete( 'jetpack_sign_in' ) }
-						completedButtonText={ translate( 'Change' ) }
-						completedTitle={ translate( 'You completed your sign in preferences.' ) }
-						description={ translate(
-							'Manage your log in preferences and two-factor authentication settings.'
-						) }
-						duration={ translate( '%d minute', '%d minutes', { count: 3, args: [ 3 ] } ) }
-						onClick={ this.handleTaskStart( {
-							taskId: 'jetpack_sign_in',
-							tourId: 'jetpackSignIn',
-							url: `/settings/security/${ siteSlug }`,
-						} ) }
-						title={ translate( 'WordPress.com sign in' ) }
-					/>
+					{ isPaidPlan && productInstallStatus && (
+						<Task
```

Comment:
As a future iteration, it might make sense to try get this into `JETPACK_CHECKLIST_TASKS` so that there's only one place where to look for these.

Reply:
Good call!